tests/integration/cmd_dev_test.go

```diff
@@ -847,9 +847,6 @@ ComponentSettings:
 
 		When("Starting a PostgreSQL service", func() {
 			BeforeEach(func() {
-				if helper.IsKubernetesCluster() {
```

Comment:
I didn't see the changes in `cmd_dev_test`, but maybe that's why the K8s tests are no longer passing for this PR?
I understand why those lines have been removed (ref #5861), and maybe the tests should be passing, but the changes in this file do not seem relevant to the original issue being addressed by this PR.
Maybe this could be done in a separate PR, to better isolate the changes.

Reply:
I don't think that is the cause of failure because the tests are passing now,but you're right about it not being relevant, I'll raise a separate PR.